docs/source/checks/vision/model_evaluation/plot_single_dataset_performance.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+.. _plot_vision_single_dataset_performance:
+
+Single Dataset Performance
+*********************************
+
+This notebooks provides an overview for using and understanding single dataset performance check.
+
+**Structure:**
+
+* `What Is the Purpose of the Check? <#what-is-the-purpose-of-the-check>`__
+* `Generate data an model <#generate-data-and-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What Is the Purpose of the Check?
+=================================
+This check returns the results from a dict of metrics, in the format metric name: ignite.Metric, calculated for the
+given model dataset.
```

Comment:
Add additional information on the required scorer type + 
what are the default scorers here (instead of below)

Reply:
@Nadav-Barak  IMO the defaults belong in the example on hoe to use with the defaults and not here. We might include a link to the metrics guide after it's complete.